x/rollapp/types/message_create_rollapp.go

```diff
@@ -48,10 +50,13 @@ func (msg *MsgCreateRollapp) ValidateBasic() error {
 	if err != nil {
 		return sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "invalid creator address (%s)", err)
 	}
-	if msg.GetMaxSequencers() == 0 {
-		return sdkerrors.Wrap(ErrInvalidMaxSequencers, "max-sequencers must be greater than 0")
-	}
 
+	if msg.GetMaxSequencers() > MaxAllowedSequencers {
+		return sdkerrors.Wrapf(ErrInvalidMaxSequencers, "is not possible to create rollapps with more than %d sequencers", MaxAllowedSequencers)
+	}
+	if uint64(len(msg.PermissionedAddresses)) > msg.GetMaxSequencers() {
+		return sdkerrors.Wrapf(ErrTooManyPermissionedAddresses, "invalid number of permissioned addresses (%d) for %d max sequencers", len(msg.PermissionedAddresses), msg.GetMaxSequencers())
+	}
```

Comment:
Maybe it's known already, but apparently `	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"` is deprecated and we should use `//	cosmossdk.io/errors` instead

Reply:
```suggestion
		return sdkerrors.Wrapf(ErrInvalidMaxSequencers, "is not possible to create rollapps with more than %d sequencers", MaxAllowedSequencers)
	}
	if uint64(len(msg.PermissionedAddresses)) > msg.GetMaxSequencers() {
		return sdkerrors.Wrapf(ErrTooManyPermissionedAddresses, "invalid number of permissioned addresses for max sequencers: n permissioned: %d: max: %d", len(msg.PermissionedAddresses), msg.GetMaxSequencers())
	}
```

Reply:
prefer making the error strings a bit more structured rather than inlining variables
it makes it easier to ctrl-f

Reply:
error msgs updated